examples/face_landmarker/ios/FaceLandmarker/Helper/OverlayView.swift

```diff
@@ -0,0 +1,142 @@
+// limitations under the License.
+
+import UIKit
+import MediaPipeTasksVision
+
+/// A straight line.
+struct Line {
+  let from: CGPoint
+  let to: CGPoint
+}
+
+/**
+ This structure holds the display parameters for the overlay to be drawon on a detected object.
+ */
+struct ObjectOverlay {
+  let dots: [CGPoint]
+  let lines: [Line]
+}
+
+/// Custom view to visualize the face landmarks result on top of the input image.
+class OverlayView: UIView {
+
+  var objectOverlays: [ObjectOverlay] = []
+  private let lineWidth: CGFloat = 3
+  private let pointRadius: CGFloat = 3
+  private let lineColor = UIColor(red: 0, green: 127/255.0, blue: 139/255.0, alpha: 1)
+  private let pointColor = UIColor.yellow
+
+  override func draw(_ rect: CGRect) {
+    for objectOverlay in objectOverlays {
+      drawDots(objectOverlay.dots)
+      drawLines(objectOverlay.lines)
+    }
+  }
+
+  /**
+   This method takes the landmarks, translates the points and lines to the current view, draws to the  of inferences.
+   */
+  func drawLandmarks(_ landmarks: [[NormalizedLandmark]], orientation: UIImage.Orientation, withImageSize imageSize: CGSize) {
+    guard !landmarks.isEmpty else {
+      objectOverlays = []
+      setNeedsDisplay()
+      return
+    }
+
+    var viewWidth = bounds.size.width
+    var viewHeight = bounds.size.height
+    var originX: CGFloat = 0
+    var originY: CGFloat = 0
+
+    if viewWidth / viewHeight > imageSize.width / imageSize.height {
+      viewHeight = imageSize.height / imageSize.width  * bounds.size.width
+      originY = (bounds.size.height - viewHeight) / 2
+    } else {
+      viewWidth = imageSize.width / imageSize.height * bounds.size.height
+      originX = (bounds.size.width - viewWidth) / 2
+    }
+
+    var objectOverlays: [ObjectOverlay] = []
+
+    for landmark in landmarks {
+      var transformedLandmark: [CGPoint]!
+
+      switch orientation {
+      case .left:
+        transformedLandmark = landmark.map({CGPoint(x: CGFloat($0.y), y: 1 - CGFloat($0.x))})
+      case .right:
+        transformedLandmark = landmark.map({CGPoint(x: 1 - CGFloat($0.y), y: CGFloat($0.x))})
+      default:
+        transformedLandmark = landmark.map({CGPoint(x: CGFloat($0.x), y: CGFloat($0.y))})
+      }
+
+      let dots: [CGPoint] = transformedLandmark.map({CGPoint(x: CGFloat($0.x) * viewWidth + originX, y: CGFloat($0.y) * viewHeight + originY)})
+      var lines: [Line] = FaceLandmarker.faceOvalConnections()
+        .map({ connection in
+        let start = transformedLandmark[Int(connection.start)]
+        let end = transformedLandmark[Int(connection.end)]
+          return Line(from: CGPoint(x: CGFloat(start.x) * viewWidth + originX, y: CGFloat(start.y) * viewHeight + originY),
+                      to: CGPoint(x: CGFloat(end.x) * viewWidth + originX, y: CGFloat(end.y) * viewHeight + originY))
+        })
+      lines.append(contentsOf: FaceLandmarker.rightEyeConnections()
```

Comment:
Can we set each connection type to be a separate color? It would really help show that each eye, lips, etc. are separate things.